fuseoverlayfs.go

```diff
@@ -494,6 +497,12 @@ func (o *snapshotter) mounts(s storage.Snapshot) []mount.Mount {
 	}
 
 	options = append(options, fmt.Sprintf("lowerdir=%s", strings.Join(parentPaths, ":")))
+	if mapping, ok := info.Labels["containerd.io/snapshot/uidmapping"]; ok {
+		options = append(options, fmt.Sprintf("uidmapping=%s", mapping))
+	}
+	if mapping, ok := info.Labels["containerd.io/snapshot/gidmapping"]; ok {
+		options = append(options, fmt.Sprintf("gidmapping=%s", mapping))
+	}
```

Comment:
@dmcgowan Label convention SGTY? Then I'd like to merge this.

Reply:
Label looks good